_posts/2016-12-21-BDD-tests-for-Hadoop-with-Cucumber-part-II.md

```diff
@@ -0,0 +1,132 @@
+---
+title: BDD tests for Hadoop with Cucumber. Part II
+description: "Tutorial how to create behaviour tests for Hadoop application with Cucumber. Part II: tests"
+layout: post
+author: bogdan
+tags: [bog data, java, hadoop, qa]
+modified: 2016-10-07
+image:
+    feature: posts/bdd_test_hadoop/title.jpg
+    
+---
+
+In [previous part](http://www.savvyclutch.com/BDD-tests-for-Hadoop-with-Cucumber-part-I/) we have created the application to process the data and docker container to isolate testing environment.
+   
+In this part we will create tests using Cucumber for Java and Gherkin.
+
+Acceptance tests, usually, check the normal behaviour of application - we want to be sure that app do what it design to do.  
```

Comment:
Acceptance tests, usually, check the normal behaviour of application - we want to be sure that _**the**_ app do what it _**is designed**_ to do.